assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -16,7 +16,7 @@ import { siteTheme } from '../../globals'
 const accessedResourceColor = siteTheme.palette.secondary.main
 const notAccessedResourceColor = siteTheme.palette.negative.main
 const linkColor = siteTheme.palette.link.main
-const mainMargin = { top: 50, right: 10, bottom: 50, left: 200 }
+const foriegnObjSide = 24
```

Comment:
```suggestion
const foreignObjSide = 24
```

Reply:
Thanks! Gosh, must have been looking at this code for too long. Should be fixed now.